src/components/AvatarGroup/AvatarGroup.visual.tsx

```diff
@@ -3,6 +3,7 @@ import { storiesOf } from '@storybook/react';
 import { AvatarGroup } from './';
 import { AvatarGroupSize } from './AvatarGroup';
 import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
```

Comment:
Oh, I guess this isn't necessary though, right?

Reply:
whops! removed